src/com/epam/javatrainings/pizzeria_homework_3/Order.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.javatrainings.pizzeria_homework_3;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.time.LocalTime;
+
+public class Order {
+    private final  int orderNumber;
+    private int customerId;
+    private List <Pizza> pizzas;
+    private LocalTime orderDateTime;
+
+    public Order (int customerId){
+        this.orderDateTime = LocalTime.now();
+        this.pizzas = new ArrayList<Pizza>();
+        this.customerId = customerId;
+        orderNumber = (int) ((Math.random() * (99999 - 10000)) + 10000);
```

Comment:
Random generation is a bad idea since it can produce the same order number for different orders, should use a sequential generation mode